sdk/core/src/main/java/org/apache/mesos/specification/DefaultService.java

```diff
@@ -82,7 +82,7 @@ public void run() {
     }
 
     private static void registerFramework(Scheduler sched, Protos.FrameworkInfo frameworkInfo, String masterUri) {
-        LOGGER.info("Registering framework: {}", frameworkInfo);
+        LOGGER.info(String.format("Registering framework: %s", frameworkInfo));
         SchedulerDriver driver = new SchedulerDriverFactory().create(sched, frameworkInfo, masterUri);
```

Comment:
Oh I believe that happened after I had committed. I kept my changes during the rebase. I'll revert. Thanks!